jmsSource/build.gradle

```diff
@@ -0,0 +1,96 @@
+group 'io.vantiq'
```

Comment:
Could we make this build log a message but not do anything if no JMS_DRIVER_LOC defined?